src/api/app/helpers/webui/patchinfo_helper.rb

```diff
@@ -4,4 +4,27 @@ def patchinfo_bread_crumb(*args)
     args.insert(0, link_to(@package, package_show_path(project: @project, package: @package)))
     project_bread_crumb(*args)
   end
+
+  def patchinfo_header(patchinfo, package_names)
+    list = package_names.to_sentence
+    text = "#{patchinfo.category} update for #{truncate(list, length: 60)}"
+    capture_haml do
+      header_title(patchinfo, text)
+      header_subtitle(patchinfo.summary, text, list)
+    end
+  end
+
+  private
+
+  def header_title(patchinfo, text)
+    haml_tag(:h3, class: 'mb-0') do
+      haml_concat(patchinfo.summary.presence || text)
+      haml_tag(:span, 'blocked', class: 'badge badge-blocked') if patchinfo.block
```

Comment:
Adding a title like `Release of this patchinfo is blocked` would make it clearer what this `blocked` means.